plan_limits.go

```diff
@@ -0,0 +1,98 @@
+//
+// Copyright 2021, Igor Varavko
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package gitlab
+
+import "net/http"
+
+// PlanLimitsService handles communication with the repositories related
+// methods of the GitLab API.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/plan_limits.html
+type PlanLimitsService struct {
+	client *Client
+}
+
+// PlanLimit represents a GitLab pipeline.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/plan_limits.html
+type PlanLimit struct {
+	PlanName                   *string `json:"plan_name,omitempty"`
+	ConanMaxFileSize           *int    `json:"conan_max_file_size,omitempty"`
+	GenericPackagesMaxFileSize *int    `json:"generic_packages_max_file_size,omitempty"`
+	HelmMaxFileSize            *int    `json:"helm_max_file_size,omitempty"`
+	MavenMaxFileSize           *int    `json:"maven_max_file_size,omitempty"`
+	NpmMaxFileSize             *int    `json:"npm_max_file_size,omitempty"`
+	NugetMaxFileSize           *int    `json:"nuget_max_file_size,omitempty"`
+	PyPiMaxFileSize            *int    `json:"pypi_max_file_size,omitempty"`
+	TerraformModuleMaxFileSize *int    `json:"terraform_module_max_file_size,omitempty"`
```

Comment:
These should all not be pointers and should be ordered alphabetically following the docs.